source/renderer/app/stores/ProfileStore.ts

```diff
@@ -293,6 +306,22 @@ export default class ProfileStore extends Store {
     return this.getTermsOfUseAcceptanceRequest.result === true;
   }
 
+  @computed
+  get areAnalyticsAccepted(): boolean {
+    return (
+      this.getAnalyticsAcceptanceRequest.result ===
+      AnalyticsAcceptanceStatus.ACCEPTED
+    );
+  }
+
+  @computed
+  get wasAnalyticsConsentScreenDisplayed(): boolean {
```

Comment:
If this a sequential flow, meaning, no two states can be `true` at the same time, I suggest refactoring status to reflect the current state.

```suggestion
  get analyticsAcceptanceStatus(): AnalyticsAcceptanceStatus
```
where `AnalyticsAcceptanceStatus` = Status.ConsentScreen || Status.Accepted || (others..?)

Reply:
Fixed, please review if you're happy with it

Reply:
Well done.